exchange_server/assets/service_checks.json

```
@@ -1 +1,33 @@
-[]
+[
+    {
+        "agent_version": "6.12.0",
```

You need to check actually when this was added, which would be when the integration was first released (check the changelog)
```suggestion
        "agent_version": "6.5.0",
```